pkg/controller/registry/reconciler/reconciler.go

```diff
@@ -319,6 +319,22 @@ func Pod(source *operatorsv1alpha1.CatalogSource, name, opmImg, utilImage, img s
 			}
 			if grpcPodConfig.ExtractContent.CacheDir != "" {
 				containerArgs = append(containerArgs, "--cache-dir="+filepath.Join(catalogPath, "cache"))
+			} else {
+				// opm serve does not allow us to specify an empty cache directory, which means that it will
+				// only create new caches in /tmp/, so we need to provide adequate write access there
+				const tmpdirPath = "/tmp/"
```

Comment:
Nit: const for `tmpdirPath` seems unnecessary since we don't use it more than once. Although a const for the volume mount name (`"tmpdir"`) would make sense since we use it in both the volume and volumeMount.